documentation/modules/mirrormaker/proc-mirrormaker-replication.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// assembly-mirrormaker.adoc
+
+[id='proc-mirrormaker-replication-{context}']
+= Synchronizing data between Kafka clusters
```

Comment:
...using Mirror Maker 2.0

?

Reply:
I did wonder how much we wanted to plaster MirrorMaker 2.0 everywhere. I'm assuming MirrorMaker 1 sections will drop at some stage, but I guess we can have it in the title for now.